src/storage/src/hummock/value.rs

```diff
@@ -126,10 +126,10 @@ impl<'a> HummockValue<&'a [u8]> {
         }
     }
 
-    /// Copies `self` into [`HummockValue<Vec<u8>>`].
-    pub fn to_owned_value(&self) -> HummockValue<Vec<u8>> {
+    /// Copies `self` into [`HummockValue<Bytes>`].
+    pub fn to_owned_bytes_value(&self) -> HummockValue<Bytes> {
```

Comment:
Bytes is design to be shared so `to_owned_bytes` seems inappropriate. Let's call it `to_bytes`.

Reply:
rename it